src/lib/helpers.ts

```diff
@@ -110,6 +111,15 @@ export function keepCurrentValueIfCompatibleDate(
   return isDate(value) || typeof value === 'string' ? value : defaultValue;
 }
 
+export function keepCurrentValueIfCompatibleRelativeDate(
```

Comment:
Should we name it whtout the "relative" part?
`keepCurrentValueIfCompatibleDate`
Because it works also for fixed dates

Reply:
keepCurrentValueIfCompatibleDate already exists and need to be keeped until we have the featue flag (because relative date should not be enabled when feature flag is set to false ;) )